docs/modeling/triggers.md

```diff
@@ -0,0 +1,113 @@
+---
+Title: Triggers
+---
+
+# Triggers
+Triggers listen to integrations and create an event when a set of defined event criteria are met. An action is then creates a payload from the event and sends it to a defined channel using [Rabbit MQ](../architecture/application.md#rabbit-mq).
+
+Examples of what triggers can be used for include: 
+
+* Starting a process instance when an email with a specific subject line is received.
+* Sending a signal to a process when a specific message is written to a Slack channel. 
+
+## Modeling triggers
+Triggers contain an [event](#events) definition that is configured in the source component such as in a connector and an [action](#actions) configured for the individual trigger. 
+
+Triggers are stored as JSON files in the format `<trigger-name>.json` and can be modeled using the GUI or the **JSON Editor**.
+
+The basic properties for triggers are: 
+
+| Property | Description | Example | 
+| -------- | ----------- | ------- | 
+| `ID` | 	The unique identifier for a trigger. This is system generated and cannot be altered. | trigger-223f6953-008a-426b-b22f-e8dd171ae47e | 
+| `Name` | The name of the trigger. Trigger names must be in lowercase and between 1 and 26 characters in length. Alphanumeric characters and hyphens are allowed, however the name must begin with a letter and end alphanumerically. | email-approval-trigger |
+| `Description` | A free text description of what the trigger does | A trigger to start a process when an approval email is received. | 
+
+### Events
+Events are created when the configured event criteria are met. 
+The properties for trigger events are: 
+
+| Property | Description | Example | 
+| -------- | ----------- | ------- | 
+| `Connector` | The [connector](../modeling/connectors/README.md) instance to use for an event | email-connector-1 | 
+| `Event` | The [connector event](../modeling/connectors/README.md#events) to use for this trigger | EMAIL_RECEIVED | 
```

Comment:
The `connector` and the `event` are not properties by itself, they are part of the `source` property.

In fact, in the next version of the triggers, there may be sources like: `rb.rb-processes-events.PROCESS_START` or `rb.rb-tasks-events.TASK_ASSIGNED` so the wat the source is created will be different than the current one in which a connector and an event are selected.

Reply:
This is a good point. I've re-worded the `source` property.